src/components/App/App.jsx

```diff
@@ -42,12 +43,15 @@ export default function App() {
 
```

Comment:
Github is not letting me comment on line 40 but since you're no longer using sampleRecipes, you can remove it.